images/admission-controller/src/aws_orbit_admission_controller/namespace.py

```diff
@@ -105,22 +120,39 @@ def process_added_event(namespace: Dict[str, Any]) -> None:
         return
 
     team_context = get_team_context(team)
-    helm_repo_url = team_context["HelmRepository"]
+    logger.info('team context keys: %s', team_context.keys())
+    helm_repo_url = team_context["UserHelmRepository"]
     logger.debug("Adding Helm Repository: %s at %s", team, helm_repo_url)
-    helm_release = f"{namespace_name}-orbit-team"
+    repo = f'{team}--userspace'
```

Comment:
is the double dashed `--userspace` intentional?

Reply:
yes.. many of our names are using - , here wanted to distinguish it that its a system postfix